lib/parzival_web/live/app/scanner_live/index.html.heex

```diff
@@ -0,0 +1,21 @@
+<div>
+  <h2 class="pt-6 text-base font-medium text-gray-900 sm:pl-6 lg:pl-8 xl:pt-6 xl:pl-6" id="announcements-title">
+    Scan an attendee's QR code
+  </h2>
+
+  <div id="loadingMessageWrapper" class="pt-5 pr-6 pb-5 pl-4 mt-16 sm:pl-6 lg:pl-8 xl:pt-6 xl:pl-6">
+    <div class="flex justify-between items-center h-10">
+      <div id="loadingMessage" class="inline-flex justify-center py-2 text-base font-medium text-gray-900">
+        Unable to access camera. Make sure you allow the use of the camera
+      </div>
+      <button onclick="requestMedia()" class="inline-flex justify-center items-center py-2 px-4 text-sm font-medium text-white rounded-md border border-transparent shadow-sm bg-secondary hover:bg-primary">Request Permission</button>
+    </div>
+  </div>
+
+  <canvas id="canvas" class="mt-16 w-full"></canvas>
+
+  <script src="/js/qrjs.js">
```

Comment:
I think we should take advantage of page specific loading as it is described in [here](https://aswinmohan.me/pagewise-js-liveview). That would allow us to transpile the JavaScript in order to make sure it works in most browsers :pray:

Reply:
Why would that be compatible with more browsers? With that link, which you had already sent me, is that the file ```qr_reading.js``` is a script more so than a module so I wasn't sure how that would be handled and this seemed easier

Reply:
You are missing the transpile step (in our case done by [esbuild](https://esbuild.github.io/)). That step is what makes us able to write modern JavaScript and not worrying about browser compatibility.